src/graphql/types/object/account-data.ts

```diff
@@ -0,0 +1,20 @@
+import { getAccountsConfig } from "@config"
+
+import { GT } from "@graphql/index"
+import { parseCustomFieldsSchema } from "@graphql/helpers"
+
+const customFieldsSchema = getAccountsConfig().customFields
+
+const AccountData = GT.Object({
```

Comment:
What happens if a new required field is introduced? Will reading even non required fields fail because it is trying to resolve required fields? Or will it not try and resolve required fields unless they are specifically requested in the query?

Reply:
we will need to setup a default value. Added here https://github.com/GaloyMoney/galoy/pull/1465/commits/7b84e3ff269f84d560fa037334943463ec3144fd

Reply:
I think doing an explicit db migration is better than having default value. 

it reduce the complexity in the code here, and the complexity to the migration, which I think is better:

those custom fields are not really meant to be dynamic and changed frequently.
when they changed, there might specific things to do on those changes, which I don't think is easy to manage from here, because each case will be different.